{"uid":"d13d59bd56711bd2","name":"test_address_prefix[Test for Imbue]","fullName":"tests.test_network_parameters.TestNetworkPrefix#test_address_prefix","historyId":"c6dfff5d7473421b217146b77ca9e91d","time":{"start":1669986361227,"stop":1669986361641,"duration":414},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7f8678c75940>\nchain = <tests.utils.chain_model.Chain object at 0x7f8678cc5700>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f8678cc5700>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Imbue\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] Imbue","statusTrace":"self = <tests.test_network_parameters.TestNetworkPrefix object at 0x7f8678c75940>\nchain = <tests.utils.chain_model.Chain object at 0x7f8678cc5700>\n\n    def test_address_prefix(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7f8678cc5700>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] Imbue\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"ec54d5fbbe428079","name":"stdout","source":"ec54d5fbbe428079.txt","type":"text/plain","size":27}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestNetworkPrefix"},{"name":"host","value":"fv-az318-139"},{"name":"thread","value":"1571-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7f8678cc5700>"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"d13d59bd56711bd2.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7f8678cc5700>"]}